.gitignore

```diff
@@ -12,9 +12,9 @@ node_modules
 /scripts/changelog-generator/token.json
 /static/dist/
 /static/fonts/
-/packages/libphonenumber/libphonenumber/libphonenumber.js
-/packages/task-utils/task-utils.js
-/ddocs/compiled.json
+/ddocs/*.json
+/ddocs/medic/_attachments
+/ddocs/medic/version
```

Comment:
Yeah good call. It means linking or copying the view files from src to dist but that's file.

I think I'll leave it as is for now because that's how the other ddocs have been working but will fix this as part of the repo layout work.